### DISCUSSION TITLE
What made Medley a great developer experience?

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#577] [Q&A]
Asked [on LinkedIn](https://www.linkedin.com/feed/update/urn:li:groupPost:830547-6866522208269938688?commentUrn=urn%3Ali%3Acomment%3A%28groupPost%3A830547-6866522208269938688%2C6867517597156458496%29)

**Comment by pamoroso** [comment]
Was the post deleted? When visiting the page I get the error `This post cannot be displayed`.

**Comment by pamoroso** [comment]
Never mind, I [asked the question again](https://groups.google.com/g/interlisp/c/aJaQIN4X3WM/m/Q8UZH0I5AQAJ) to Medley Interlisp Users/Interest so that there's a permanent record.
